WooCommerce/Classes/ViewRelated/Upgrades/UpgradesView.swift

```diff
@@ -6,7 +6,6 @@ import Yosemite
 /// To be used to display available current plan Subscriptions, available plan Upgrades,
 /// and the CTA to upgrade
 ///
-@MainActor
```

Comment:
@iamgabrielma I removed the @MainActor here because it seems unused, please feel free to check again.